src/components/Board.js

```diff
@@ -16,10 +16,71 @@ class Board extends Component {
     };
   }
 
+  componentDidMount() {
+    axios.get('https://inspiration-board.herokuapp.com/boards/ultra_princess/cards')
+      .then((res) => {
+        this.setState({ cards: res.data })
+      })
+      .catch((error) => {
+        alert('hm ur cards didnt load maybe u should refresh lol');
+      });
+  }
+
+  delCard = (id) => {
+    const url = 'https://inspiration-board.herokuapp.com/cards/' + id.toString();
+
+    axios.delete(url)
+      .then(() => {
+        let index;
+        let newState = this.state;
+
+        newState.cards.forEach((card, i) => {
+          if (card.card.id === id) {
+            index = i;
+          }
+        });
+
+        newState.cards.splice(index, 1);
+
+        this.setState(newState);
+      })
+      .catch((e) => {
+        alert('sry could not delete ?? idk tbh..');
+        console.log(e);
+      });
+  }
+
+  addCard = (newCard) => {
+    const url = `https://inspiration-board.herokuapp.com/boards/ultra_princess/cards?text=${newCard.text}&emoji=${newCard.emoji}`;
+
+    axios.post(url)
+      .then((res) => {
+        let newState = this.state;
```

Comment:
I like that you've kept all the API interaction logic in one place - the callbacks are a little more complex, but I would say it makes the app as a whole much easier to comprehend. Whether or not you intended it, this is a great example of the container component pattern well-applied.